src/pool/accounts.json

```diff
@@ -1,5 +1,92 @@
 [
-  "0x00686735619287df0f11ec4cda22675f780886b52bf59cf899dd57fd5d5f4cad",
```

Comment:
the issue is that we need to be able to select between staging and testnet here

Reply:
What do you think about merging this as is considering only staging for now and then tracking in a follow up to add an attribute on each inside the JSON to have either testnet or staging?

Reply:
ok sounds good